File: pset6/mario.py
import cs50

def main():

    #prompt user for an integer between 1 and 
    while True:
        print("Height: ", end = "")
        n = cs50.get_int()
        
        if n > 1 or n < 23:
            break
        
    for i in range(n):
        for j in range(n-i-1):
            print(" ", end ="")
        for k in range(i+1):
            print("#", end="")
            
        print("  ", end="")
        
        for l in range(i+1):
            print("#", end="")
        print("")
        
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        printf("Height: ");
        n = get_int();
        if (n == 0)
        {
            return 0;
        }
    }while (n < 1 || n > 23);
    
    for (int i = 0; i < n; i++)
    {
        for (int j = i + 1; j < n; j++)
        {
            printf(" ");
        }
        for (int j = n - 1 - i; j < n; j++)
        {
            printf("#");
        }
        
        printf("  ");
        
        for (int j = n - 1 - i; j < n; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}